File: 3636-check-balanced-string/check-balanced-string.py
class Solution:
    def isBalanced(self, num: str) -> bool:
        num_arr = [int(n) for n in num]
        sum_even = sum(num_arr[i] for i in range(len(num_arr)) if i % 2 == 0)
        sum_odd = sum(num_arr) - sum_even
        return sum_even == sum_odd

File: 3636-check-balanced-string/check-balanced-string.js
/**
 * @param {string} num
 * @return {boolean}
 */
var isBalanced = function(num) {
    let oddSum = 0;
    let evenSum = 0;
    for (let i = 0; i < num.length; i++) {
        if (i % 2 !== 0) {
            oddSum += parseInt(num[i]);
        }
        else {
            evenSum += parseInt(num[i]);
        }
    }
    return oddSum === evenSum;    
};